app/controllers/bundle_contexts_controller.rb

```diff
@@ -5,6 +5,9 @@ def index
 
   def create
     @bundle_context = BundleContext.new(bundle_contexts_params)
```

Comment:
I know it isn't the point of this PR, but `MyController#create` should probably do `MyModel.create` and not have to specify template explicitly.  (I think that is the default rails-magic behavior.)

Reply:
talking w/ Justin C. and looking at some of the rails docs, its more conventional to do the `.new` then `.save`

Reply:
I disagree with you here.  Despite the prevalence of new/save in intro tutorials, there is literally no advantageous difference, since create (w/o an Array) just does:

https://github.com/rails/rails/blob/master/activerecord/lib/active_record/persistence.rb#L31-L39
```ruby
object = new(attributes, &block)
object.save
object
```
And we can tell from `.persisted?` or `.errors` whether the save was successful or not.